docs/book/src/tasks/healthcheck.md

```diff
@@ -109,6 +109,7 @@ Before deploying a MachineHealthCheck, please familiarise yourself with the foll
 - If the Node for a Machine is removed from the cluster, a MachineHealthCheck will consider this Machine unhealthy and remediate it immediately
 - If no Node joins the cluster for a Node after the `NodeStartupTimeout`, the Machine will be remediated
 - If a Machine fails for any reason (if the FailureReason is set), the Machine will be remediated immediately
+- If a machine has `cluster.x-k8s.io/paused` or `cluster.x-k8s.io/skip-remediation` annotation, it will not be considered for remediation
```

Comment:
can we add more detailed docs on how this is meant to be used from a user perspective? I assume the "paused" annotation will be added automatically as part of clusterctl move whereas the "skip-remediation" annotation is meant to be added by users to arbitrarily skip remediation for certain machines?